devops/ci/multi_node_deploy/3node_deploy.jenkinsfile

```diff
@@ -199,6 +199,11 @@ node {
                 echo "Successfully deployed system states!"
             }
 
+            stage("utils Software Deployment") {
+                sshCommand remote: remotes[0], command: "provisioner deploy_vm --states utils --setup-type 3_node"
+                echo "Successfully deployed utils states!"
```

Comment:
"Successfully deployed utils states!"
to
"Successfully deployed foundation states!"